src/utils/helpers.js

```diff
@@ -0,0 +1,61 @@
+import bcrypt from 'bcrypt';
+import jwt from 'jsonwebtoken';
+
+const salt = bcrypt.genSaltSync(10);
+const secret = process.env.JWTSECRET;
+/**
+ * Helper class
+ */
+export default class Helper {
+  /**
+   * Method to exclude properties from an object
+   * @param {object} objectItem - The object to exclude fields from
+   * @param {array} fields - Array of fields to be removed
+   * @returns {object} - new object
+   */
+  static omitFields(objectItem, fields) {
+    const items = objectItem;
+
+    fields.forEach((field) => {
+      delete items[field];
+    });
+
+    return items;
+  }
+
+  /**
+   * Method to exclude properties from an object
+   * @param {object} objectItem - The object to extract fields from
+   * @param {array} fields - Array of fields to be extracted
+   * @returns {object} - new object
+   */
+  static pickFields(objectItem, fields) {
+    const items = {};
+
+    fields.forEach((field) => {
+      items[field] = objectItem[field];
+    });
+
+    return items;
+  }
+
+  /**
+   * Method encrypt data
+   * @param {*} data - params data
+   * @returns { string} - encypted data
+   */
+  static encryptor(data) {
+    const encrypted = bcrypt.hashSync(data, salt);
```

Comment:
Why not use async like await bcrypt.hash(data, salt) before hashSync is blocks the thread

Reply:
Here is an example to make it dynamic
So that this method job is solely for encryption instead of declaring the salt outside

static async encryptor(data, rounds) {
    const salt = await bcrypt.genSalt(rounds)
    const encrypted = await bcrypt.hash(data, salt);

    return encrypted;
  }